### DISCUSSION TITLE
FarmPi on other SBCs than RasPi?

### DISCUSSION THREAD

**Discussion by KalixtShawxo** [mkevenaar/FarmPi#19] [Q&A] [answered]
Since the OctoFarm requires a 64bit OS and hardware, and is not using RASPIAN itself as a base, could the FarmPi image be put on to say a hardware sufficient Odroid or PINE64 SBC? 
I had installed OctoFarm manually on an Ubuntu system using an old Dell Desktop, then I did FarmPi on a RasPi 3... but now I'm wondering about moving it to another SBC entirely. Just using the FarmPi image versus doing a manual install on the other SBC after putting on a suitable OS. 

Feasible? Or should I just leave it on the RasPi 3?

**Comment by mkevenaar** [comment, marked as answer]
Hi @KalixtShawxo 

Sorry for the late reply, I did not see this discussion coming in.

I have heard people trying the FarmPi image on different boards without success.

Basically all devices shown for Ubuntu Server shown on [this](https://ubuntu.com/download/raspberry-pi) page that support 64 bit are supported.

You can try it on your own board, but there would be no support as I don't have access nor funds to get those boards to test them with and support you.
